src/copy.jl

```diff
@@ -144,3 +144,12 @@ end
 function Base.deepcopy(::Model)
     error("`JuMP.Model` does not support `deepcopy` as the reference to the underlying solver cannot be deep copied, use `copy` instead.")
 end
+
+function MOI.copy_to(dest::MOI.ModelLike, src::Model)
+    bridged_dest = MOI.Bridges.full_bridge_optimizer(dest, Float64)
```

Comment:
What if `dest` is already bridged ? There cannot be multiple layers of bridges because variables bridges assume that the  underlying model does not return variable with negative indices.